lib/Model/IMAPMessage.php

```diff
@@ -327,9 +327,6 @@ private function hasAttachments($part) {
 			/** @var Horde_Mime_Part $p */
 			$filename = $p->getName();
 
-			if ($p->getContentId() !== null) {
```

Comment:
what if we instead check the content-disposition header?

Reply:
why not indeed?
It's what I did in the frontend in https://github.com/nextcloud/mail/pull/4667/files

Reply:
This would cause an issue when the message's body is of type text/plain: in such case, the inline attachment won't be shown

Reply:
Don't know if it's that important though...

Also, maybe we should simply replace this function by Horde_Mime_Part->isAttachments() ? As what's done in MessageMapper...

Reply:
This becomes too much of a change. I'll create another PR to provide a quick fix to #4758.

Here I'll try to make various changes to improve how we handle attachment and inline mime parts.

Is it possible to cancel the backport of this PR?